README.md

```diff
@@ -1,76 +1,100 @@
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-## 🤝 Contributing
+<p align="center">
+  <a href="https://www.microverse.org/">
+    <img alt="Microverse" src="https://img.shields.io/badge/-Microverse-blueviolet?style=flat-square">
+  </a>
+  <a href="https://github.com/Hamzaoutdoors/to-do-list/blob/development/LICENSE">
+    <img alt="MIT Licensed" src="https://img.shields.io/github/license/Hamzaoutdoors/to-do-list?style=flat-square">
+  </a>
+  <a href="https://github.com/Hamzaoutdoors/js_capstone">
+    <img alt="GitHub last commit (branch)" src="https://img.shields.io/github/last-commit/Hamzaoutdoors/js_capstone/dev?color=blue&style=flat-square">
+  </a>
+  <a href="https://github.com/Hamzaoutdoors/js_capstone">
+    <img alt="GitHub Repo stars" src="https://img.shields.io/github/stars/Hamzaoutdoors/js_capstone?color=green&label=%E2%98%85%20stars%20&style=flat-square">
+  </a>
+  <a href="https://github.com/Hamzaoutdoors">
+    <img alt="GitHub followers" src="https://img.shields.io/github/followers/Hamzaoutdoors?color=yellow&logo=github&style=flat-square">
+  </a>
+</p>
+
+<div align="center">
+ <img src="https://img.shields.io/badge/javascript-%23323330.svg?style=for-the-badge&logo=javascript&logoColor=%23F7DF1E"/> <img src="https://img.shields.io/badge/html5-%23E34F26.svg?style=for-the-badge&logo=html5&logoColor=white"/> <img src="https://img.shields.io/badge/css3-%231572B6.svg?style=for-the-badge&logo=css3&logoColor=white"/> <img src="https://img.shields.io/badge/bootstrap-%23563D7C.svg?style=for-the-badge&logo=bootstrap&logoColor=white"/> <img src="https://img.shields.io/badge/git-%23F05033.svg?style=for-the-badge&logo=git&logoColor=white"/> <img src="https://img.shields.io/badge/webpack-%238DD6F9.svg?style=for-the-badge&logo=webpack&logoColor=black"/> <img src="https://img.shields.io/badge/-jest-%23C21325?style=for-the-badge&logo=jest&logoColor=white"/></div>
+
+# JS-Capstone: Series lover
+
+> The Series lover website displays scores submitted by different series. It also allows users to submit their comments in every serie show. All data is preserved thanks to the external [TVmaze API](https://www.tvmaze.com/api).
+ 
+ <p align="center">
+    <img alt="Screenshot" src="https://user-images.githubusercontent.com/80895497/132874077-4edf055c-eb84-4326-9d7c-f5aa67fec688.png"width="700" height="300">
+</p>
+<p align="center">
+    <img alt="Popup" src="https://user-images.githubusercontent.com/80895497/132874290-3cfe25ba-9f64-45dc-8066-359eb212e6b2.png"width="700" height="300">
+    <p align="center">screen shot of our project</p>
+</p>
+
+# Capstone Live Demo
+[Live Demo Link](https://hamzaoutdoors.github.io/js_capstone/)
+
+## Video presentation
+
+[![Video](https://user-images.githubusercontent.com/80895497/132873016-3cb89f3c-f0d3-4d69-b48f-e43810653a23.png)](https://drive.google.com/file/d/1nxTdAebrj8g08sjhaXpOBEbDju1GlYig/view?usp=sharing)
```

Comment:
- [ ] Great presentation you guys provided but I believe that @Emmanuelaaron voice kept cutting out several times. 🙂 For me! It was ok and I understood the whole point, but for future viewers! They may not get the best impression about it 🙏 😄 Also, it seems that @Hamzaoutdoors haven't got the chance to present through the app. So, I highly recommend recording another presentation ❤️ and this time try to cover more about a piece of code that you guys are proud of making. 😉

Reply:
For this, we had an internet issue that's why I had present most of the time other than @Hamzaoutdoors

Reply:
Thanks for your feedback Mahmoud we appreciate it